src/peers/ui.py

```diff
@@ -86,6 +94,19 @@ def effect_removed(self, event_data):
         else:
             print(f"Tried to remove effect but couldnt find it")
 
+    def log_message(self, level: str, timestamp: float, origin: str, message: str):
+        self._logs.append({
```

Comment:
This means self._logs will keep growing until manually cleared, right? Should we set a stupidly large maximum amount, as a precaution?

Reply:
Yes I thought about sending events only with the newly added log and the entire list to make sure that even if the list grows large we dont send massively large websocket packets. Perhaps I should actually add that AND set a maximum limit?

Reply:
Sounds good